### DISCUSSION TITLE
Instrument sounds

### DISCUSSION THREAD

**Discussion by dblnk** [powertab/powertabeditor#450] [Q&A] [answered]
Hi, I quite like the sound of the acoustic bass in the power tab editor. 
Is there a way to extract the sound engine for that particular instrument and import it into a DAW such as Ableton?

**Comment by cameronwhite** [comment, marked as answer]
The instrument sounds are just the MIDI instruments, so if you check what the `MIDI Output Device` is set to in the `Preferences` window you could probably use the same device from your DAW for MIDI content? On Windows for examples, this is probably the default Windows software synth

**Reply by dblnk** [reply]
Thanks for the prompt response! In my case on macOS Big Sur, the MIDI Output Device is set to "Power Tab Software Synth". I haven't been able to locate that device in the Power Tab Editor "Package contents" in Finder. Any idea where it is stored?

**Reply by cameronwhite** [reply]
Ah right, on macOS it's a weird case where PTE just starts up a virtual midi device which plays back using the AudioUnit synth. So there isn't a separate executable, it's just started in the background when launching the editor
From a bit of googling http://blog.synthesizerwriter.com/2017/01/using-apple-general-midi-dls-sound-bank.html might let you do something equivalent in Ableton?
